frontend/src/pages/BYONImages/BYONImagesTable.tsx

```diff
@@ -105,42 +89,39 @@ export const BYONImagesTable: React.FC<BYONImagesTableProps> = ({ images, forceU
     return total;
   };
 
-  const getSortableRowValues = (nb: BYONImage): string[] => {
-    const { name, description = '', phase = '', visible = false, user = '', uploaded = '' } = nb;
-    return [name, description, phase, visible.toString(), user, uploaded.toString()];
-  };
-
-  if (activeSortIndex !== undefined) {
-    [...images].sort((a, b) => {
-      const aValue = getSortableRowValues(a)[activeSortIndex];
-      const bValue = getSortableRowValues(b)[activeSortIndex];
-
-      if (activeSortDirection === 'asc') {
-        return (aValue as string).localeCompare(bValue as string);
-      }
-      return (bValue as string).localeCompare(aValue as string);
-    });
-  }
-
-  const getSortParams = (columnIndex: number): ThProps['sort'] => ({
-    sortBy: {
-      index: activeSortIndex,
-      direction: activeSortDirection,
-      defaultDirection: 'asc',
+  const columns: SortableData<BYONImage>[] = [
+    {
+      field: '',
+      label: '',
+      sortable: false,
     },
-    onSort: (_event, index, direction) => {
-      setActiveSortIndex(index);
-      setActiveSortDirection(direction);
+    {
+      field: 'name',
+      label: 'Name',
+      sortable: (a, b) => a.name.localeCompare(b.name),
     },
-    columnIndex,
-  });
-
-  const columnNames = {
-    name: 'Name',
-    description: 'Description',
-    user: 'User',
-    uploaded: 'Uploaded',
-  };
+    {
+      field: 'description',
+      label: 'Description',
+      sortable: (a, b) => (a.description as string).localeCompare(b.description ?? ''),
```

Comment:
Yep. The BYON Table component is pretty old and does not follow most of the same table design as the other tables. I figure this will be revamped later down the road. 

But yeah i just copied the same logic from the old table, but i can change it to use `??` now